packages/story-editor/src/components/layout/index.js

```diff
@@ -60,11 +60,10 @@ const Editor = withOverlay(styled.section.attrs({
   width: 100%;
   display: grid;
   grid:
-    'lib   canv        insp' 1fr
-    'lib   supplementary insp' auto /
-    minmax(${LIBRARY_MIN_WIDTH}px, ${LIBRARY_MAX_WIDTH}px)
-    minmax(${CANVAS_MIN_WIDTH}px, 1fr)
-    minmax(${INSPECTOR_MIN_WIDTH}px, ${INSPECTOR_MAX_WIDTH}px);
+    'insp   canv          ' 1fr
```

Comment:
There's a number of lint warnings in this file. Is husky not working currently? I actually feel like I haven't seen it running for a while.

Reply:
We updated it recently; you might need to run `npx husky-init` once

Reply:
Done in https://github.com/GoogleForCreators/web-stories-wp/pull/10760/commits/b85b713e56b824ebc646ad5e7f7ae12935cde0eb